MANIFEST.in

```diff
@@ -1,5 +1,10 @@
 include setup_boilerplate.py
 include *requirements.txt
+include pyproject.toml
 include LICENSE
 include NOTICE
 include ./*/py.typed
+recursive-include test *.py
```

Comment:
Rather than including everything blindly, I'll add a rule for all __init__.py, and for examples.py separately. If setuptools starts ignoring tests altogether, I'll probably reconsider the whole idea of using setuptools to package my software...